src/stylesheets/components/_inventory.scss

```diff
@@ -0,0 +1,21 @@
+.inventory--MAIN {
+  background-color: $grey-1;
+  overflow: auto;
+
+  .inputfield--MAIN {
+    margin: 1.5rem 0 0 1.5rem;
```

Comment:
yup! Again, just habits I've picked up for responsive design / CSS. I like to keep things in rems because they're relative to the font-size at root (as opposed to ems which are relative to their parent, so can get out of control quickly)